kernel/src/main/java/org/kframework/utils/Tag.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2012-2020 K Team. All Rights Reserved.
+package org.kframework.utils;
+
+public class Tag {
+
+    public final static String require = "require";
```

Comment:
Let's not bring this whole file back when we're just using it to mark the constant strings that are part of the Bubble class. Just put them as static fields on that class.